pkg/cmd/admin/groups/new.go

```diff
@@ -72,6 +82,41 @@ func (o *NewGroupOptions) Complete(f *clientcmd.Factory, args []string) error {
 	}
 
 	o.GroupClient = osClient.Groups()
+
+	outputFormat := kcmdutil.GetFlagString(cmd, "output")
+	templateFile := kcmdutil.GetFlagString(cmd, "template")
+	printer, _, err := kubectl.GetPrinter(outputFormat, templateFile)
+	if err != nil {
+		return err
+	}
+
+	if printer != nil {
+		o.Printer = printer.PrintObj
+	} else {
+		o.Printer = func(_ runtime.Object, out io.Writer) error {
```

Comment:
By doing something like [this](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/create/user.go#L86-L88)?  Or something else?